starter_kits/Python3/hlt/positionals.py

```diff
@@ -62,10 +63,17 @@ def invert(direction):
 
 
 class Position:
-    def __init__(self, x, y):
+    def __init__(self, x, y, normalize=True):
```

Comment:
I think we should always normalize. Performance penalty is trivial compared to all the other horrible things python has.